protostar-rust/src/main.rs

```diff
@@ -0,0 +1,107 @@
+use anyhow::{anyhow, Context, Result};
+use cairo_lang_protostar::test_collector::LinkedLibrary;
+use clap::Parser;
+use rust_test_runner_bindings::run_tests;
+use scarb_metadata::{Metadata, MetadataCommand, PackageId, PackageMetadata};
+use serde::Deserialize;
+use std::env::set_var;
+use std::path::PathBuf;
+
+const CONTRACT_TARGET: &str = "starknet-contract";
+
+#[derive(Parser, Debug)]
+struct Args {
+    // #[clap(required = true)]
+    // paths: Vec<PathBuf>,
+    #[arg(short, long)]
+    linked_libraries: Option<Vec<PathBuf>>,
+}
+
+#[derive(Deserialize, Debug)]
+struct ProtostarConfig {
+    exit_first: Option<bool>,
+    ignore: Option<Vec<String>>,
+    json: Option<bool>,
+    last_failed: Option<bool>,
+    report_slowest_tests: Option<bool>,
```

Comment:
I don't think there is a need to support any of those right now, you can add support for `exit-first` only as a PoC of integration

Reply:
These are not handled anywhere atm. They will be worked on later after PoC is done - there are tasks for them already.

Reply:
> They will be worked on later after PoC is done
I think it should be more distant future. Keeping them here may be misleading for other developers

Reply:
We have to defined some kind of `ProtostarTestConfig` to test if reading protostar config from scar.toml works at all so I don't think why it shouldn't already be a mostly correct struct.

Reply:
I have seen situations in my career when such interfaces are defined and many people are certain for a long period of time that those things are already implemented. Structure definition is part of dev docs, we should make it represent how things are.

Reply:
I am removing those for now, if you have strong opinion about this (I have :p) we can discuss it later